src/ext/BalExtension/wixstdba/wixstdba.mc

```diff
@@ -0,0 +1,73 @@
+; // Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
+
+
+MessageIdTypedef=DWORD
+
+LanguageNames=(English=0x409:MSG00409)
+
+
+; // message definitions
+
+; // MessageId=#
+; // Severity=Success
+; // SymbolicName=MSG_SUCCESS
+; // Language=English
+; // Success %1.
+; // .
+;
+; // MessageId=#
+; // Severity=Warning
+; // SymbolicName=MSG_WARNING
+; // Language=English
+; // Warning %1.
+; // .
+;
+; // MessageId=#
+; // Severity=Error
+; // SymbolicName=MSG_ERROR
+; // Language=English
+; // Error %1.
+; // .
+
+MessageId=1
```

Comment:
They'll only show up if the BAFunctions was loaded.  It's supposed to be for the same reason that we do it in the engine - log what WixStdBA would have done so when someone complains about weird behavior we can point to the line in the log where their code changed something.